sanctuary/sanctuary-types.d.ts

```diff
@@ -54,6 +54,7 @@ type NonZeroFiniteNumber = number; // - Infinity, -Infinity, 0, -0, NaN
 type ValidNumber = number; // - NaN
 
 export interface Functor<F> {
+  map<U>(fn: (f: F) => U): Functor<U>;
```

Comment:
Should this be `fantasy-land/map` rather than `map`, to match the method name? (I don't know TypeScript.)